src/wp-content/themes/twentytwelve/css/editor-blocks.css

```
@@ -18,53 +18,64 @@ Description: Used to style blocks in the editor.
 /*--------------------------------------------------------------
 1.0 General Typography
 --------------------------------------------------------------*/
-
+.editor-styles-wrapper,
+.editor-styles-wrapper p,
```

```suggestion

.editor-styles-wrapper,
```

This should have a blank line between the comment and the ruleset.

Also, the paragraph would not need the font size if `.editor-styles-wrapper` has it, so I removed the `.editor-styles-wrapper p` line.